### DISCUSSION TITLE
Strategy Meeting Notes

### DISCUSSION THREAD

**Discussion by xames3** [ciesin-geospatial/TOPSTSCHOOL-SPHINX#34] [Thoughts and Feedbacks]
Hello hello!

I'm adding the meeting notes that were discussed during a strategy planning meeting with @kyttmacmanus, @jfmartinez4, @antoniotovargh, @xames3 and @AlexandrBig1.

@antoniotovargh, mentioned:
- Leveraging the Sphinx framework for bite-sized lessons.
- Advocating open science through means of videos, engagement processes and social media. Inviting people interested in collaborating to the team meetings.
- More community-driven lessons.
- Develop mini lessons using smaller datasets, fine-tuning Jupyter notebooks to stream necessary data.

Other suggestions shared by @AlexandrBig1 and @xames3 were:
- Incorporating Google Analytics for tracking web-traffic and SEO.
- Custom domain name for the sites. Shared an example on the email.
- Reliance on GitHub Discussion Boards for more transparency.
- Incorporating [DUTC](https://www.dontusethiscode.com/index.html)'s open science related videos as part of tutorials.

**Comment by Alexandrbig1** [comment]
Hi Team,

Following up on our recent strategy planning meeting, I’ve written a blog post about how the meeting went and the key ideas we discussed.

You can read it here: https://paragraph.xyz/@alexsmagin/nasa-tops-school-team-meeting

Feel free to share it with your networks to help spread the word about our work and invite others to join our open science mission.

Thanks again for the great collaboration—I’m excited about what’s ahead!

**Reply by xames3** [reply]
where's the link?

**Comment by Alexandrbig1** [comment]
Wow Alex, this is awesome!! I will definitely share it around.

**Comment by Alexandrbig1** [comment]
Terrific, thank you Alex and happy holidays everyone!

Best,
Kytt

On Wed, Nov 27, 2024 at 12:37 PM Christina Deodatis <
***@***.***> wrote:

> Wow Alex, this is awesome!! I will definitely share it around.
>
> On Wed, Nov 27, 2024 at 12:23 PM Alex Smagin ***@***.***>
> wrote:
>
>> Hi *Team*,
>>
>> Following up on our recent strategy planning meeting, I’ve written a
>> *blog* post about how the meeting went and the key ideas we discussed.
>>
>> You can *read it here*:
>> ***@***.***/nasa-tops-school-team-meeting
>>
>> Feel free to *share* it with your networks to help spread the word about
>> our work and invite others to join our *open science mission*.
>>
>> Thanks again for the great collaboration—I’m excited about what’s ahead!
>>
>> Best regards,
>> *Alex Smagin*
>>
>> *Full Stack Developer*
>>
>> My Portfolio <https://www.alexsmagin.dev>
>>
>> On Sun, Nov 24, 2024 at 7:16 PM XA ***@***.***> wrote:
>>
>>> Hello hello!
>>>
>>> I'm adding the meeting notes that were discussed during a strategy
>>> planning meeting with @kyttmacmanus <https://github.com/kyttmacmanus>,
>>> @jfmartinez4 <https://github.com/jfmartinez4>, @antoniotovargh
>>> <https://github.com/antoniotovargh>, @xames3 <https://github.com/xames3>
>>> and @Alexandrbig1 <https://github.com/Alexandrbig1>.
>>>
>>> @antoniotovargh <https://github.com/antoniotovargh>, mentioned:
>>>
>>>    - Leveraging the Sphinx framework for bite-sized lessons.
>>>    - Advocating open science through means of videos, engagement
>>>    processes and social media. Inviting people interested in collaborating to
>>>    the team meetings.
>>>    - More community-driven lessons.
>>>    - Develop mini lessons using smaller datasets, fine-tuning Jupyter
>>>    notebooks to stream necessary data.
>>>
>>> Other suggestions shared by @Alexandrbig1
>>> <https://github.com/Alexandrbig1> and @xames3
>>> <https://github.com/xames3> were:
>>>
>>>    - Incorporating Google Analytics for tracking web-traffic and SEO.
>>>    - Custom domain name for the sites. Shared an example on the email.
>>>    - Reliance on GitHub Discussion Boards for more transparency.
>>>    - Incorporating DUTC <https://www.dontusethiscode.com/index.html>'s
>>>    open science related videos as part of tutorials.
>>>
>>> —
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/ciesin-geospatial/TOPSTSCHOOL-SPHINX/discussions/34>,
>>> or unsubscribe
>>> <https://github.com/notifications/unsubscribe-auth/AZWU42VJA644F6VKGGBI7ND2CJ3AJAVCNFSM6AAAAABSM3FDAOVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZXGU2DOOJRGM>
>>> .
>>> You are receiving this because you were mentioned.Message ID:
>>> ***@***.***>
>>>
>>

-- 
Kytt MacManus (he/him)
Assistant Systems Engineer, NASA Socioeconomic Data and Applications Center
(SEDAC)
GEO-Human Planet Initiative co-lead
GIS Developer, CIESIN | Columbia Climate School
Adjunct Lecturer, School of International and Public Affairs
Adjunct Lecturer, E3B Dept of Columbia College

PO Box 1000, 61 Rt 9W
Palisades NY 10964 USA
V: 845-365-8939 F: 845-365-8922
https://www.ciesin.columbia.edu
https://sedac.ciesin.columbia.edu

**Comment by Alexandrbig1** [comment]
This is great! Thank you Alex! Happy Thanksgiving everyone!

Camilla

**Comment by Alexandrbig1** [comment]
Happy Thanksgiving everyone! ❤️
